deployment/website.yaml.tmpl

```diff
@@ -70,4 +70,6 @@ spec:
             - name: FLASK_ENV
               value: <FLASK_ENV>
             - name: GCS_BUCKET
-              value: <GCS_BUCKET>
\ No newline at end of file
+              value: <GCS_BUCKET>
+            - name: PROJECT
```

Comment:
project is very generic -- do you mean GCP project here?
Maybe we can add comments to the yaml.tmpl files?

Reply:
renamed to SECRET_PROJECT. I like this suggestion as I once have to remember what project this is.

Reply:
👍